app/controllers/songs_controller.rb

```diff
@@ -1,10 +1,25 @@
 class SongsController < ApplicationController
   def index
+    @songs = Song.all
```

Comment:
Wait a minute: why do we have both the index and search actions? They can be just one method. Let's just call it index so move the search code into index.

Also, `Song.all` can probably be `Song.order(:name)`.